app/controllers/coronavirus_form/data_export_controller.rb

```diff
@@ -43,7 +43,7 @@ def usage_statistics(start_date, end_date)
 private
 
   def produce_csv(results)
-    csv_data = CSV.generate do |csv|
+    csv_data = CSV.generate(col_sep: "|") do |csv|
```

Comment:
We will need to make the downstream consumers of this data (whoever they are?) aware of this.